src/logger/console-logger.service.ts

```diff
@@ -21,7 +21,7 @@ export class ConsoleLoggerService {
     this.classContext = context;
   }
 
-  error(message: any, trace = '', functionContext?: string) {
+  error(message: unknown, trace = '', functionContext?: string) {
     this.printMessage(
       message,
       clc.red,
```

Comment:
`ESLint: Unsafe member access .magentaBright on an any value.(@typescript-eslint/no-unsafe-member-access)`

Maybe we should import the colors directly instead of importing clc?
`import { red } from 'cli-color';`

Reply:
That should be gone. Did you run `yarn install`?

Reply:
I did and this still occurs. Curiously it does not happen when I run `yarn lint`, but only as an error in the IDE.

Reply:
Maybe delete your `.eslintcache`?

Reply:
The backend doesn't have a `.eslintcache`or at least I don't…